cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -417,7 +417,10 @@ def test_cell_type_ontology_term_id(self, validator_with_adata, term):
         validator = validator_with_adata
         validator.adata.obs.loc[validator.adata.obs.index[0], "cell_type_ontology_term_id"] = term
         validator.validate_adata()
-        assert validator.errors == [f"ERROR: '{term}' in 'cell_type_ontology_term_id' is not allowed."]
+        # Forbidden columns may be marked as either "not allowed" or "deprecated"
+        assert validator.errors == [
```

Comment:
this test goes through all forbidden `cell_type_ontology_term_id` terms and previously asserted that these all came up as "X is not allowed". it seems like some of these now show up as "X is a deprecated term id of 'CL'". i don't think it's important for our tests to assert which CL terms are marked as "not allowed" or "deprecated", but let me know if that assumption is wrong.